product_attribute_archive/views/product_attribute.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Akretion
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="product_attribute_form_view">
+        <field name="name">product.attribute.form</field>
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.product_attribute_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='main_fields']" position="before">
+                    <field name="active" invisible="1" />
+                    <widget
+                    name="web_ribbon"
+                    title="Archived"
+                    bg_color="bg-danger"
+                    attrs="{'invisible': [('active', '=', True)]}"
+                />
+            </xpath>
+            <field name="active" optional="hide" />
```

Comment:
Unable to show column active on product.attribute form, you should apply to place that column inside the Attribute Value tree

```suggestion
            <xpath expr="//field[@name='value_ids']/tree" position="inside">
              <field name="active" optional="hide"/>
            </xpath>
```

Reply:
Rebase required to fix pre-commit